eth/client.go

```diff
@@ -571,41 +572,50 @@ func (c *client) GetTranscoderTokenPoolsForRound(addr common.Address, round *big
 func (c *client) GetDelegator(addr common.Address) (*lpTypes.Delegator, error) {
 	dInfo, err := c.BondingManagerSession.GetDelegator(addr)
 	if err != nil {
+		glog.Infof("Error getting delegator from bonding manager: %v", err)
 		return nil, err
 	}
 
 	dStatus, err := c.DelegatorStatus(addr)
 	if err != nil {
+		glog.Infof("Error getting status: %v", err)
 		return nil, err
 	}
 
 	status, err := lpTypes.ParseDelegatorStatus(dStatus)
 	if err != nil {
 		return nil, err
 	}
-
-	pendingStake, err := c.PendingStake(addr)
+	currentRound, err := c.CurrentRound()
 	if err != nil {
+		glog.Infof("Error getting current round: %v", err)
 		return nil, err
 	}
 
-	pendingFees, err := c.PendingFees(addr)
-	if err != nil {
+	pendingStake, err := c.PendingStake(addr, currentRound)
+	if err != nil && err.Error() != "abi: unmarshalling empty output" {
```

Comment:
I think ignoring the "abi: unmarshalling empty output" error here can create issues for the caller of this function since it won't be able to differentiate between the scenario in which the pendingStake returned is actually 0 and the scenario in which the pendingStake returned is 0 because there was a function call failure (likely due to the function call not completing due to there being too many rounds to iterate through). Maybe we still return the error here and in the CLI we check for the error and if the error is "abi: unmarshalling empty output" the CLI displays a message notifying the user that the pendingStake cannot be calculated since there are too many rounds to iterate through

Reply:
Hmm... But we still have all the other parameters that might be useful for people.

What do you think about setting `pendingStake` and `pendingFee` to be -1 if we see `abi:unmarshalling empty output`?  This basically means "we can't tell what's going on".

Reply:
Right right, I'd be on board with setting `pendingStake` and/or `pendingFee` to be -1 if the error is `abi: unmarshalling empty output` and in lieu of a number value for those field in the interface, display a message to notify the user that those values cannot be computed with a single function call. However, they could be computed by the client by looping through the rounds and querying each round's claimable rewards/fees - this would require many RPC requests though which would be inefficient so perhaps we can just go with the single function call with the user notification message in the event of failure for the time being